parameter/AreaConfiguration.cpp

```diff
@@ -125,52 +128,46 @@ bool CAreaConfiguration::serializeXmlSettings(CXmlElement& xmlConfigurableElemen
 }
 
 // Compound handling
-const CConfigurableElement* CAreaConfiguration::getConfigurableElement() const
+const CConfigurableElement *CAreaConfiguration::getConfigurableElement() const
 {
     return _pConfigurableElement;
 }
 
-void CAreaConfiguration::copyToOuter(CAreaConfiguration* pToAreaConfiguration) const
+void CAreaConfiguration::copyToOuter(CAreaConfiguration *pToAreaConfiguration) const
 {
     assert(_pConfigurableElement->isDescendantOf(pToAreaConfiguration->getConfigurableElement()));
 
-    copyTo(&pToAreaConfiguration->_blackboard, _pConfigurableElement->getOffset() - pToAreaConfiguration->getConfigurableElement()->getOffset());
+    copyTo(&pToAreaConfiguration->_blackboard,
+           _pConfigurableElement->getOffset() -
+               pToAreaConfiguration->getConfigurableElement()->getOffset());
 }
 
-void CAreaConfiguration::copyFromOuter(const CAreaConfiguration* pFromAreaConfiguration)
+void CAreaConfiguration::copyFromOuter(const CAreaConfiguration *pFromAreaConfiguration)
 {
     assert(_pConfigurableElement->isDescendantOf(pFromAreaConfiguration->getConfigurableElement()));
 
-    copyFrom(&pFromAreaConfiguration->_blackboard, _pConfigurableElement->getOffset() - pFromAreaConfiguration->getConfigurableElement()->getOffset());
+    copyFrom(&pFromAreaConfiguration->_blackboard,
+             _pConfigurableElement->getOffset() -
+                 pFromAreaConfiguration->getConfigurableElement()->getOffset());
 
     // Inner becomes valid
     setValid(true);
 }
 
-CParameterBlackboard& CAreaConfiguration::getBlackboard()
-{
-    return _blackboard;
-}
+CParameterBlackboard &CAreaConfiguration::getBlackboard() { return _blackboard; }
```

Comment:
I get that we want this for methods defined in headers (for concision) but it is against our coding style to do it in source files. Is there a way to selectively turn it on?